reference.py

```diff
@@ -0,0 +1,114 @@
+import datajoint as dj
+
+schema = dj.schema(dj.config['names.%s' % __name__], locals())
+
+
+@schema
+class User(dj.Manual):
```

Comment:
Quite clearly, this table is used as backend table for Alyx system and thus many of the fields do not necessary make sense as scientific schema. We should discuss how should "users" in the sense of the system user should (or should not) be separated from "users" in the sense of experimenters.

Reply:
true, this could be django baggage (users are application level), leaving as is for now pending further discussion